ecf/scripts/gdas/atmos/obsproc/prep/jgdas_atmos_emcsfc_sfc_prep.ecf

```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:08:00
-#PBS -l select=1:ncpus=1:mem=2GB
+#PBS -l select=1:mpiprocs=80:ompthreads=1:ncpus=80:mem=4GB
```

Comment:
Please revert the cores.  Memory increase is ok.

Reply:
Reverted